packages/react-next/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -345,6 +345,12 @@ function useDismissHandlers(
       }
     };
 
+    const dismissOnTargetWindowBlur = (ev: FocusEvent) => {
+      if (!preventDismissOnLostFocus && !targetWindow?.document.hasFocus() && ev.relatedTarget === null) {
```

Comment:
I just checked in a fix for this, but there was another property added. This can't be the default behavior as it breaks embedded scenarios in things like Iframes.